radio/src/translations/de.h

```diff
@@ -39,18 +39,18 @@
 #define TR_VBLMODE                     "AUS","Taste","Stks","Beide","EIN"
 #define TR_TRNMODE                     "AUS","+=",":="
 #define TR_TRNCHN                      "CH1","CH2","CH3","CH4"
-#define TR_AUX_SERIAL_MODES            "AUS","Telem Mirror","Telemetry In","SBUS Eingang","LUA","CLI","GPS","Debug","SpaceMouse"
+#define TR_AUX_SERIAL_MODES            "AUS","Telem Spiegel","Telemetrie In","SBUS Eingang","LUA","CLI","GPS","Debug","SpaceMouse"
```

Comment:
"Spiegel" is a bit too literal IMO, I would not get the meaning as a newcomer.
How about "weiterl."?

Concerning the telemetry in, AFAIK it currently only supports FrSky Hub telemetry in, so could we just call it "FrSky Hub In" until other protocols are supported?

Reply:
yes, you are right. I will change to "weiterl." It's a good Idea I think.